src/transaction/atomic_replication_helper.cpp

```diff
@@ -487,29 +578,32 @@ namespace cublog
       case RVHF_MVCC_UPDATE_OVERFLOW:
       case RVHF_INSERT_NEWHOME:
       {
-	assert (m_watcher_p == nullptr);
-	m_watcher_p.reset (new PGBUF_WATCHER ());
+	assert (watcher_uptr == nullptr);
```

Comment:
I think `RVHF_UPDATE_NOTIFY_VACUUM` should be added as a case here too.

Reply:
Done